public/js/gamecore.js

```diff
@@ -62,10 +63,23 @@ $(document).ready(function()  {
                     for (var shot in attacksMade[attack]) {
                         // getting each towerID, creepID pair
                         // this is why we must send stuff as array of objects...
-                        drawShot(gameCtx, attack, attacksMade[attack][shot]);
+                        // drawShot(gameCtx, attack, attacksMade[attack][shot]);
+
+                        projectiles.push(
+                            new Projectile(gameCtx,
+                                attack,
+                                attacksMade[attack][shot],
+                                15));
```

Comment:
Yeah, this definitely isn't the way to do this in the future.  I left it in so the backend can start integrating the port into their stuff and use the front end side to see how changing parameters changes the movement of the projectile while a communication protocol is set up.